src/js/containers/IpRewriteCard/IpRewriteCard.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { FormattedMessage, injectIntl } from 'react-intl';
+import { asyncPluginUpdateSetting } from '../../actions/ipRewrite';
+import { Card, CardSection, CardContent, CardControl, CardDrawers } from 'cf-component-card';
+import Toggle from 'cf-component-toggle';
+
+const SETTING_NAME = "ip_rewrite";
+
+class IpRewriteCard extends Component {
+
+    handleChange(value) {
+        let { activeZoneId, dispatch } = this.props;
+        dispatch(asyncPluginUpdateSetting(SETTING_NAME, activeZoneId, value));
+    }
+
+    render() {
+        const { formatMessage } = this.props.intl;
+        return (
+            <div>
+                <Card>
+                    <CardSection>
+                        <CardContent title={formatMessage({id: 'container.ipRewrite.title'})}>
+                            <p><FormattedMessage id="container.ipRewrite.description" /></p>
+                        </CardContent>
+                        <CardControl>
+                            <Toggle
+                                label=""
+                                value={(this.props.ipRewriteValue)}
+                                onChange={this.handleChange.bind(this)}/>
+                        </CardControl>
+                    </CardSection>
+                </Card>
+            </div>
+        );
+    }
+}
+
+function mapStateToProps(state) {
+    return {
+        activeZoneId: state.activeZone.id,
+        ipRewriteValue: state.pluginSettings.entities[state.activeZone.id][SETTING_NAME].value,
```

Comment:
We need to move this check into a selector.  Create src/js/selectors/ipRewrite.js:
```
function getPluginSettingsForZoneId(zoneId, state) {}
function getPluginSettingsValueForZoneId(zoneId, settingId, state) {}
``
The first one we'll add to PerformancePage.js to ensure the plugin settings have loaded before we render the card.
The second one will be the value of ipReWriteValue.
```
ipRewriteValue: getPluginSettingsValueForZoneId(state.activeZone.id, SETTING_NAME, state)
```
We duplicate these state checks across multiple pages/cards and need to keep it as consistent as possible because the shape of our state tree will likely change once WWW finishes migrating to React/Redux and we're able to use more of their code.